src/web-auth/captcha.js

```diff
@@ -2,18 +2,21 @@
 import Authentication from '../authentication';
 import object from '../helper/object';
 
-var noop = function() {};
+var noop = function () {};
 
 var RECAPTCHA_V2_PROVIDER = 'recaptcha_v2';
 var RECAPTCHA_ENTERPRISE_PROVIDER = 'recaptcha_enterprise';
 var HCAPTCHA_PROVIDER = 'hcaptcha';
 var FRIENDLY_CAPTCHA_PROVIDER = 'friendly_captcha';
+var ARKOSE_PROVIDER = 'arkose';
 var AUTH0_PROVIDER = 'auth0';
+var TIMEOUT_MS = 500;
+var MAX_RETRY = 3;
```

Comment:
Should we make it clear these values are only for arkose?

Reply:
I could. But our hope is that have retry logic for all captchas at some point in the future. Plus in ULP, we used the same naming as here. Do you think we should make the name specific to Arkose?